.vscode/extensions.json

```diff
@@ -0,0 +1,16 @@
+{
+  "recommendations": [
+    "GraphQL.vscode-graphql",
+    "ms-azuretools.vscode-docker",
+    "dbaeumer.vscode-eslint",
+    "esbenp.prettier-vscode",
+    "rvest.vs-code-prettier-eslint",

```

Comment:
I haven't used this one. Is there much added value on top of the official extension (esbenp.prettier-vscode)?

Reply:
Not required to be used, my opinion